oops/backplane/unittester.py

```diff
@@ -1,289 +1,58 @@
 ################################################################################
 # oops/backplane/unittester.py
 ################################################################################
-
-from __future__ import print_function
-
-import numpy as np
-import os.path
-
-from polymath import Vector3
-
-from oops.backplane    import Backplane
-from oops.body         import Body
-from oops.event        import Event
-from oops.meshgrid     import Meshgrid
-from oops.surface.ansa import Ansa
-import oops.config as config
-from oops.backplane.exercise_backplanes import exercise_backplanes
-
-
-################################################################################
-# UNIT TESTS
-################################################################################
-
+#
+#usage: unittester.py [-h] [--args [arg [arg ...]]] [--verbose]
+#                     [--diff old new] [--exercises-only] [--no-exercises]
+#                     [--no-compare] [--output dir] [--no-output] [--log]
+#                     [--undersample N] [--reference] [--test-level N]
+#
+# optional arguments:
+#  -h, --help            show this help message and exit
+#  --args [arg [arg ...]]
+#                        Generic arguments to pass to the test modules. Must
+#                        occur last in the argument list.
+#  --verbose             Print output to the terminal.
+#  --diff old new        Compare new and old backplane logs.
+#  --exercises-only      Execute only the backplane exercises.
+#  --no-exercises        Execute all tests except the backplane exercises.
+#  --no-compare          Do not compare backplanes with references.
+#  --output dir          Directory in which to save backplane PNG images.
```

Comment:
So does this mean that every time someone runs the tests, it regenerates the PNG files and updates the files in Dropbox? That seems bad. The work done by a developer should stay local to their workstation and only update the global dropbox when specifically requested to. In fact if a person has read-only access to the folder, like would happen on an test server, the directory won't even be writeable. Perhaps there needs to be a different default destination directory.